redirect/routes.php

```diff
@@ -225,11 +225,11 @@ class Route {
         ],
         'gad'              => [
             'description' => 'Grundlegende Algorithmen und Datenstrukturen',
-            'target'      => 'https://artemis.ase.in.tum.de/courses/119',
+            'target'      => 'https://artemis.ase.in.tum.de/courses/184',
         ],
         'gadunittests'     => [
             'description' => 'Unit - Tests: Grundlegende Algorithmen und Datenstrukturen',
-            'target'      => 'https://github.com/Code-Connect/TUM_Homework/tree/master/src/gad17/',
+            'target'      => 'https://github.com/MaximilianAnzinger/gad2022-tests',
```

Comment:
Currently, this is not yet as helpful as the other repo. Did they completely change the tasks making the old tests obsolete? If not it might be a nice idea to also link to https://github.com/Code-Connect/TUM_Homework/tree/master/src/gad17 from your repo until it has some content.

Reply:
I'm one of the students who is reworking the exercises. And yes, not even these tests (https://github.com/N0W0RK/TUM_GAD_Tests_SS21) from last year are very helpful.
However, I can certainly add both of them as inspiration for new tests.

Reply:
Thanks for clarifying :)